tools/mtmd/mtmd-audio.cpp

```diff
@@ -486,11 +486,11 @@ bool mtmd_audio_preprocessor_whisper::preprocess(
     params.hann_window_size = hparams.audio_window_len;
     params.hop_length       = hparams.audio_hop_len;
     params.sample_rate      = hparams.audio_sample_rate;
-    params.center_padding   = false;
-    params.preemph          = 0.0f; // disabled
-    params.use_natural_log  = false;
-    params.norm_per_feature = false;
-    params.need_chunking    = true;
+    params.center_padding   = hparams.audio_center_padding;
+    params.preemph          = hparams.audio_preemph; // disabled
+    params.use_natural_log  = hparams.audio_use_natural_log;
+    params.norm_per_feature = hparams.audio_norm_per_feature;
+    params.need_chunking    = hparams.audio_need_chunking;
```

Comment:
probably better to remove this `need_chunking` as it is not actually a filter param

instead, we should better create new class extending from `mtmd_audio_preprocessor` as explained in https://github.com/ggml-org/llama.cpp/pull/17978#discussion_r2618874901